oci/layer/generate_linux_test.go

```diff
@@ -32,7 +32,7 @@ func TestInsertLayerTranslateOverlayWhiteouts(t *testing.T) {
 	}
 
 	testNode := path.Join(dir, "test")
-	err = unix.Mknod(testNode, unix.S_IFCHR|0666, int(unix.Mkdev(0, 0)))
+	err = system.Mknod(testNode, unix.S_IFCHR|0666, unix.Mkdev(0, 0))
```

Comment:
Ah, you need to add an `import` for `system` -- add `"github.com/opencontainers/umoci/pkg/system` to the imports block at the top of the file.